src/cdk/a11y/live-announcer/live-announcer.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('LiveAnnouncer', () => {
     afterEach(() => {
       // In our tests we always remove the current live element, in
       // order to avoid having multiple announcer elements in the DOM.
-      announcer.ngOnDestroy();
+      announcer.clear();
```

Comment:
I think we actually want to call `ngOnDestroy` here to remove the live-element as it could otherwise leak to other unit tests.

Reply:
Right. Done. With clear, at least the innerText will be cleared (if not the live-element), which (in our case) is good enough.